lego/apps/events/serializers/events.py

```diff
@@ -227,8 +227,8 @@ class Meta(EventReadSerializer.Meta):
             "unregistered",
             "waiting_registrations",
             "use_consent",
-            "share_info_flag",
-            "created_by",
+            "use_contact_tracing",
+            "created_by",UG
```

Comment:
:thinking:

Reply:
You still think we only should have the user id, and not the entire public user?

Reply:
Ooo, my typo. Yeah, I only use this to compare the user id with the user who created the event.